spec/index.bs

```diff
@@ -1975,6 +1975,7 @@ enum GPUTextureFormat {
     "bgra8unorm",
     "bgra8unorm-srgb",
     // Packed 32-bit formats
+    "rgb9e5float",
```

Comment:
Thinking about this further, I'm not sure if we should call this a "float" format, or even a packed format. It isn't a packed format in the way rg11b10float is, and the components aren't individually floats.

From the realization I had in the call - this is in some way a compressed texture format (with a block size of 1x1). (Compression is cheap, unlike BC/ETC, but also lossy with multiple possible compression results.) Maybe we should just call it "rgb9e5" and put it just before the BC format list?

Reply:
> It isn't a packed format in the way rg11b10float is,

What do you mean? I think it's packed in a similar way into 32 bits.

> the components aren't individually floats.

They *are* individually floats. Each component has 9 mantissa and 5 exponent bits. The fact that the exponent is shared doesn't make them non-floats.

Reply:
> What do you mean? I think it's packed in a similar way into 32 bits.

I think it's arguably different because the components are interdependent. That doesn't normally happen except with compressed formats, so in that way it's like a compressed format.

> They _are_ individually floats. Each component has 9 mantissa and 5 exponent bits. The fact that the exponent is shared doesn't make them non-floats.

Actually, I think it's slightly different from the others (I was going to look this up yesterday but forgot). IIUC rgb9e5 doesn't omit an implicit leading "1." the way ieee754 does. But of course it's still *a* floating-point format.

Reply:
I do agree with the idea that the format has something in common with compressed formats. Yet, we shouldn't be sucked into this idea, and instead follow what the native APIs do. Both Vulkan and Metal call it a "float" format, it's not ambiguous. D3D12 also describes it as float:
> Three partial-precision floating-point numbers encoded into a single 32-bit value all sharing the same 5-bit exponent (variant of s10e5, which is sign bit, 10-bit mantissa, and 5-bit biased (15) exponent).

Reply:
I don't like trying to categorize it as a "float" format if it's just for the sake of being able to categorize it. It's a unique format, and I don't think it hurts to give it a unique name, and even D3D12 (which I think generally does the best job of naming) agrees (though I think "rgb9e5" without any qualifier is good enough). OTOH we do use unorm/snorm/ufloat/sfloat in the names of our compressed texture formats so we should be consistent with those.

With that in mind, rgb9e5 is actually a `ufloat` format. Should we call it `rgb9e5ufloat`?

Reply:
Yes, `rgb9e5ufloat` is good

Reply:
hrm, just looked it up and rg11b10 is also ufloat.

Reply:
+1 for using `rgb9e5ufloat` and renaming to `rg11b10ufloat`.